apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx

```diff
@@ -18,6 +18,7 @@
 
 import { SBACInterface, TestableComponentInterface } from "@wso2is/core/models";
 import { Heading, InfoCard, useMediaContext } from "@wso2is/react-components";
+import { identityProviderConfig } from "apps/console/src/extensions";
```

Comment:
Better to include the relative path like in other places.

Reply:
Fixed